quickstarts/microsoft.documentdb/cosmosdb-create-multi-region-account/main.bicep

```diff
@@ -30,30 +30,30 @@ param api string = 'Sql'
 ])
 param defaultConsistencyLevel string = 'Session'
 
-@description('Max stale requests. Required for BoundedStaleness. Valid ranges, Single Region: 10 to 1000000. Multi Region: 100000 to 1000000.')
+@description('Max stale requests. Required for BoundedStaleness. Valid ranges, Single Region: 10 to 2,147,483,647. Multi Region: 100,000 to 2,147,483,647.')
```

Comment:
I suspect you used the commas for readability ```100,000``` but there will be users who put in the commas for the values and that will fail...

Reply:
Yes, I did. All of the updates I added commas. You've already merged some of these. Do you want me to submit another PR to remove?

I also need your help in figuring out a path for this empty parameter. I've tried a bunch of different things including passing an empty string into Capabilities or just removing it altogether. Not sure how to get this to work.

Reply:
re: the commas, I don't think that's urgent, I suspect the UX in portal will flag them, so it's mostly frustrating in that case...

re: the empty param maybe ping offline for that one.